table/block_based/block_based_table_reader.cc

```diff
@@ -3052,6 +3110,8 @@ Status BlockBasedTable::Prefetch(const Slice* const begin,
 }
 
 Status BlockBasedTable::VerifyChecksum() {
+  // TODO(haoyu): This function is called by external sst ingestion and user.
```

Comment:
External sst ingestion is also called by user.

Reply:
I updated the comments.